src/main/java/Command.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * The Command class makes use of multiple classes to drive the program.
+ */
+public class Command {
+
+    private String command;
+
+    /**
+     * Creates a Command object.
+     *
+     * @param command A string representation of the user's command.
+     */
+    public Command(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Responds accordingly to the user's command.
+     *
+     * @param tasks A TaskList object containing the list of tasks.
+     * @param ui An Ui object that helps to interact with the user.
+     * @param storage A Storage object that contains a file with the tasks in it.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
As per Ellieyee's comment below, you could also extract and make abstractions for each of the individual commands.